### PR TITLE
您好开发人员

### PR DESCRIPTION
**Pull request by Yumingyuan**
结合您在issues中提出的需求，我结合样例实现了signature example，并创建了一个dapp_examples文件夹以便放入这些样例，烦请抽空检查。我的bounty账号为:timdavid@vip.qq.com

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-portal/164)

### CODE REVIEW

**Comment by yqrashawn** [comment]
Corresponding bounty https://bounty.conflux-chain.org/view-bounty?bountyId=5ee20cddac7af7652002f1dd

Corresponding issue #152

File: dapp_examples/js-eth-personal-sign-examples/index.js [hunk 1]
```diff
@@ -0,0 +1,437 @@
+var ethUtil = require('ethereumjs-util')
+var sigUtil = require('eth-sig-util')
+var Eth = require('ethjs')
+window.Eth = Eth
```
Comment (yqrashawn) [inline dapp_examples/js-eth-personal-sign-examples/index.js:4]: 你好, 这里做 demo 的目的是让后续的 dapp 开发者有相关代码可以参考. 我们通过 https://github.com/Conflux-Chain/js-conflux-sdk 提供了 js api, 这里使用的还都是以太坊的库.

**Comment by Yumingyuan** [comment]
> Builds ready [[9129532](https://github.com/Conflux-Chain/conflux-portal/commit/91295320eca1afbfdc8eadf022d70cf059a576ad)]
> * builds: [chrome](https://5852-228328387-gh.circle-artifacts.com/0/builds/conflux-portal-chrome-0.2.2.zip), [firefox](https://5852-228328387-gh.circle-artifacts.com/0/builds/conflux-portal-firefox-0.2.2.zip), [opera](https://5852-228328387-gh.circle-artifacts.com/0/builds/conflux-portal-opera-0.2.2.zip)
> * bundle viz: [background](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/background.html), [ui](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/ui.html), [inpage](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/inpage.html), [contentscript](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/contentscript.html), [ui-libs](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/ui-libs.html), [bg-libs](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/bg-libs.html), [phishing-detect](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/source-map-explorer/phishing-detect.html)
> * dep viz: [background](https://5852-228328387-gh.circle-artifacts.com/0/build-artifacts/deps-viz/background/index.html)
> * [all artifacts](https://circleci.com/gh/Conflux-Chain/conflux-portal/5852#artifacts/containers/0)

？？？

File: dapp_examples/js-eth-personal-sign-examples/index.js (hunk 1 above)
Comment (angelia-yuqi-personal) [reply dapp_examples/js-eth-personal-sign-examples/index.js:4]: 可以参考这个repo [sign-demo](https://github.com/danfinlay/js-eth-personal-sign-examples)，但这是基于以太坊的api，要使用我们 [conflux sdk](https://github.com/Conflux-Chain/js-conflux-sdk) 提供的 js api

Comment (Yumingyuan) [reply dapp_examples/js-eth-personal-sign-examples/index.js:4]: 已参考API完成，感觉有一些API缺失了

**Comment by Yumingyuan** [comment]
https://github.com/Conflux-Chain/conflux-portal/issues/152

**Comment by Yumingyuan** [comment]
您好，我已修正昨日问题，并更新github，请检查

**Comment by Yumingyuan** [comment]
https://github.com/Yumingyuan/Portal_dapp/issues/3

**Pull request closed without merging**